python/metatensor_operations/metatensor/operations/_dispatch.py

```diff
@@ -377,6 +377,21 @@ def make_contiguous(array):
         raise TypeError(UNKNOWN_ARRAY_TYPE)
 
 
+def column_add(output_array, input_array, index):
```

Comment:
Hmm, we might want to unify this with the existing `_dispatch.index_add`. I'll look into it.